src/uu/test/src/parser.rs

```diff
@@ -208,7 +228,7 @@ impl Parser {
 
             // case 2: error if end of stream is `( <any_token>`
             [symbol] => {
-                show_error!("missing argument after ‘{:?}’", symbol);
+                show_error!("missing argument after ‘{}’", symbol);
```

Comment:
Note that GNU uses slightly different quotes with `LANG=C`, which is what we currently try to match until we have localization support.